packages/eip-1193-provider/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@turnkey/eip-1193-provider",
+  "version": "0.0.1",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js",
+      "default": "./dist/index.mjs"
+    }
+  },
+  "types": "./dist/index.d.ts",
+  "license": "Apache-2.0",
+  "description": "EIP-1193 Provider for Turnkey.",
+  "scripts": {
+    "version": "node -p \"'export const VERSION = ' + JSON.stringify(require('./package.json').name + '@' + require('./package.json').version) + ';'\" > src/version.ts",
```

Comment:
~~I don't think this is relevant for this package? This + the version.ts file can probably go unless you think there's a use-case.~~

(the reason why we have this in the http package is to make sure version can be passed as a header to downstream requests)

EDIT: disregard this, it _is_ used!